src/store/index.ts

```diff
@@ -1,13 +1,21 @@
 import {configureStore} from "@reduxjs/toolkit";
+import decodeReducer from "./header/decodeJwtSlice";
 import headerReducer from "./header/headerSlice";
+import linksReducer from "./links/linksSlise";
 import loginReducer from "./loginName/loginSlice";
 import useReducer from "./loginName/userSlice";
```

Comment:
naming, 
 `useReducer` ->  `userReducer`?
What `userSlice` describes, and how is it different from JwtSlice which describes the user session and also contains the user name? (ideally, it should be clear from the name of the slice)

Reply:
userSlice checks Login is unicum or not
JwtSlice gets token from localStorage and decode it.